skunkworks_crow/src/main/java/org/odk/share/views/ui/main/MainActivity.java

```diff
@@ -203,22 +224,18 @@ private boolean isCollectInstalled() {
     private void showAlertDialog() {
         AlertDialog.Builder builder = new AlertDialog.Builder(this);
         builder.setMessage(R.string.install_collect);
-        builder.setPositiveButton(getString(R.string.install), new DialogInterface.OnClickListener() {
-            @Override
-            public void onClick(DialogInterface dialog, int which) {
-                try {
-                    startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("market://details?id=" + COLLECT_PACKAGE)));
-                } catch (android.content.ActivityNotFoundException e) {
-                    startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://play.google.com/store/apps/details?id=" + COLLECT_PACKAGE)));
-                }
+
+        builder.setPositiveButton(getString(R.string.install), (DialogInterface dialog, int which) -> {
+            try {
+                startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("market://details?id=" + COLLECT_PACKAGE)));
+            } catch (android.content.ActivityNotFoundException e) {
```

Comment:
Please import the class at the top of the class instead of using fully qualified name.

Reply:
You mean, static import? Done.